app/config/routing.yml

```diff
@@ -5,8 +5,8 @@
 # the routes in YAML, XML or PHP files.
 # See http://symfony.com/doc/current/book/routing.html
 app:
-    resource: '@AppBundle/Controller/'
-    type:     annotation
+    resource: '@AppBundle/Action/'
```

Comment:
Yes I've opened an alternative PR for that. But I think I'll do it even in the default setup of the bundle. It will be more familiar for Symfony developers (and it seems to be a general concern).